src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -779,7 +804,10 @@ void GeneralPage::initLayout(GeneralPageView &layout)
             [](auto args) {
                 return args.index + 1;
             },
-            false);
+            false,
+            "Customizes how you see Asian Language names.\ne.g. "
+            "testaccount_420(테스트계정420) -> tesaccount_420\n"
+            "testaccount_420(테스트계정420) -> 테스트계정420");
```

Comment:
Could you add the option value names to this? Not sure if users would know what a localized name is, maybe this could be better explained?